src/System.Net.Requests/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uap-Windows_NT;
```

Comment:
could you have some comment here telling removing the UAP configuration after fixing the issue #20136 ?